back/engines/commercial/id_nemlog_in/app/lib/id_nemlog_in/feature_specification.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'citizen_lab/mixins/feature_specification'
+
+module IdNemlogIn
+  module FeatureSpecification
+    extend CitizenLab::Mixins::FeatureSpecification
+
+    def self.feature_name
+      'nemlog_in_login'
```

Comment:
Technically this isn't really needed as we are not planning to enable this through the main login screen for Copenhagen, only through verified actions, but it's in here anyway